netplugin/plugin/netplugin_test.go

```diff
@@ -195,28 +198,3 @@ func TestNetPluginInitInvalidConfigMissingNetworkDriverName(t *testing.T) {
 		t.Fatalf("plugin init succeeded, should have failed!")
 	}
 }
-
-func TestNetPluginInitInvalidConfigMissingFwdMode(t *testing.T) {
```

Comment:
Removing this test as its no longer valid. https://github.com/contiv/netplugin/blob/0f843fa92f923f6f37378c3956ae02ab3855350e/netplugin/plugin/netplugin.go#L299 there's always a fwd mode.